src/lib/components/image/_index.scss

```diff
@@ -0,0 +1,268 @@
+.vuiImage {
+  &__imageWrapper {
+    position: relative;
+    display: inline-block;
+    line-height: 0;
+  }
+
+  &__image {
+    display: block;
+    max-width: 100%;
+    height: auto;
+    transition: opacity $transitionSpeed;
+
+    &--loaded {
+      opacity: 1;
+    }
+
+    &--clickable {
+      cursor: pointer;
+    }
+  }
+
+  &__captionOverlay {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    padding: $sizeM $sizeM;
+    background: rgba(0, 0, 0, 0.7);
+    color: $colorEmptyShade !important;
+  }
+
+  &__previewOverlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: $sizeS;
+    background: rgba(0, 0, 0, 0.7);
+    backdrop-filter: blur(8px);
+    color: $colorEmptyShade;
+    cursor: pointer;
+    opacity: 0;
+    transition: opacity $transitionSpeed;
+
+    svg {
+      color: $colorEmptyShade;
+    }
+
+    p {
+      color: $colorEmptyShade;
+    }
+  }
+
+  &__imageWrapper:hover &__previewOverlay {
+    opacity: 1;
+  }
+
+  &__error {
+    background-color: rgba($colorDanger, 0.05);
+    border: 1px solid rgba($colorDanger, 0.2);
+    border-radius: $sizeXxs;
+    min-height: 150px;
+  }
+
+  // Size variants
+  &--xs {
+    .vuiImage__image {
+      max-width: 100px;
+      max-height: 100px;
+    }
+
+    .vuiImage__imageWrapper {
+      max-width: 100px;
```

Comment:
Do we need any of the max-width and max-height styles set on `vuiImage__imageWrapper`? I think we can remove them all. For that matter is the `vuiImage__imageWrapper` class needed?